src/BasketBundle/Controller/BasketController.php

```diff
@@ -522,7 +524,19 @@ public function finalReviewStepAction()
 
         return $this->render('SonataBasketBundle:Basket:final_review_step.html.twig', [
             'basket' => $basket,
-            'tac_error' => 'POST' == $this->get('request')->getMethod(),
+            'tac_error' => 'POST' == $this->getRequest()->getMethod(),
         ]);
     }
+
+    /**
+     * @return Request
+     */
+    public function getRequest()
```

Comment:
Can't we make those private?

Reply:
No, this method is public in parent class in SF 2.8. But we can rename it, make private and replace all usages of getRequest().

Reply:
Not sure if it is a good idea to name it like the parent, because on SF 3.4 I think it is removed or final?

Maybe give it another name like: `getCurrentRequest`?

Reply:
Done